pkg/apply/desiredset_process.go

```diff
@@ -346,23 +346,38 @@ func (o *desiredSet) list(namespaced bool, informer cache.SharedIndexInformer, c
 	)
 
 	if informer == nil {
```

Comment:
is this not a problem too when we have an informer and use the cache below?

Reply:
The informer cache behavior wasn't changed as part of any of the previous PRs (https://github.com/rancher/wrangler/pull/183, https://github.com/rancher/wrangler/pull/187). It appears to only function with either non-namespaced objects, or when a namespace is provided. I'm assuming that matches its use since none of the controllers that use it with an informer were broken by the previous changes.

Reply:
lgtm then